<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="UserLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="UserLabel">
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="UserNameInput">
        <property name="minimumSize">
         <size>
          <width>171</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="PasswordlLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="PasswordLabel">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PasswordInput">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="ButtonLayout">
      <item>
       <widget class="QPushButton" name="RegisterButton">
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LogInButton">
        <property name="text">
         <string>Log in</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
